charts/cloudzero-agent/docs/cicd-chart-testing.md

```diff
@@ -19,19 +18,17 @@ Here is an example output:
 
 ```bash
 Stage  Job ID                                        Job name                                      Workflow name             Workflow file                 Events           
-0      build-and-publish-chart                       build-and-publish-chart                       build-and-publish-chart   build-and-publish-chart.yml   push             
-0      has_changes                                   has_changes                                   build_test_publish_image  build-test-publish-image.yml  push,pull_request
-0      scanner                                       scanner                                       detection_rules           change-detector.yml           workflow_call    
-1      test_chart_lint                               test_chart_lint                               build_test_publish_image  build-test-publish-image.yml  pull_request,push
-1      build_test_chart_install_maybe_publish_image  build_test_chart_install_maybe_publish_image  build_test_publish_image  build-test-publish-image.yml  push,pull_request
+0      build-and-publish-chart  build-and-publish-chart  build-and-publish-chart  build-and-publish-chart.yml  push             
```

Comment:
nit: these don't line up with the headers of this table, at least, viewing it in github rich diff they looked misaligned.

Reply:
good eye! fixing in current push